modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -229,25 +229,33 @@
 		}
 	}
 
-	.e-con,
 	.e-n-tabs-content {
-		border: 1px none var( --n-tabs-border-color );
-		border-width: var( --n-tabs-border-width );
+
+		:where( & > .e-con ) {
```

Comment:
Why are we using `:where()`. Is that for specificity? Is that necessary for the Nested tabs? 

I would consider just using `> e.con` instead. 

What are your thoughts?

Reply:
using the `:where()` in all the places where we set styling for the `.e-con` are necessary so that when someone chooses to style the drop down Container itself, that the long chain of specific selectors in the Widget here don't override the selector of the Container.

and the same goes for the `'selector' => '...'` in the Controls.

This was the solve for the problem in Mega Menu where the Widget Container controls were overriding any custom dropdown Container controls. and we decided to apply the same logic to the Tabs.